### ISSUE TITLE
Support local API

### ISSUE THREAD

**Issue by vadim0x60** [urschrei/pyzotero#191]
Since Zotero 7, every installation of Zotero runs a local clone of the web API. In theory, a user that has Zotero installed could speed up `pyzotero` significantly with just one line of code:

```
zot = zotero.Zotero(library_id, library_type, api_key)
zot.endpoint = 'http://localhost:23119/api/'
```

Unfortunately, the way `pyzotero` builds URLs breaks this solution: for some reason the `/api` part of the endpoint gets dismissed and `pyzotero` tries calling `http://localhost:23119/users/` instead of `http://localhost:23119/api/users/`.

With a little rework of the `build_url` function one could simplify the code as well as enable the ability to use local API.

**Comment by urschrei** [comment]
Sounds good, feel free to open a PR.

**Comment by urschrei** [comment]
Now available on pypi (v1.5.27): set `local=True` when initialising your Pyzotero instance.
